### DISCUSSION TITLE
A typo on the GitHub Copilot settings page

### DISCUSSION THREAD

**Discussion by gauravmak** [community/community#19559] [Other Feature Feedback, Questions, & Ideas] [answered]
_Not sure if this is the right place to report this. Please guide accordingly._

![image](https://user-images.githubusercontent.com/11887260/175967294-1542a665-c433-4b78-9a2e-e5fbb1f16cce.png)

**Comment by melvincwng** [comment, marked as answer]
@gauravmak, you can report bugs to [GitHub Bug Report](https://support.github.com/contact/bug-report) over here.

Hope this clarifies.

**Reply by melvincwng** [reply]
@gauravmak, if this reply was helpful, I would appreciate it if you can kindly 'Mark as answer'.

Thank you, and have a great day ahead 😄
